examples/platforms/simulation/trel.c

```diff
@@ -473,6 +476,8 @@ void platformTrelProcess(otInstance *aInstance, const fd_set *aReadFdSet, const
     }
 }
 
+const otPlatTrelCounters *otPlatTrelGetCounters() { return &sCounters; }
```

Comment:
Particularly important to have `(void)` in C files, otherwise C compiler will allow any arguments.
```suggestion
const otPlatTrelCounters *otPlatTrelGetCounters(void) { return &sCounters; }
```

Reply:
Done